scripts/collect-unresolved-breaches-addresses.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+// TODO: Confirm db row has index
+
+const Knex = require("knex");
+const knexConfig = require("../db/knexfile");
+const knex = Knex(knexConfig);
+
+const HIBP = require("../hibp");
+
+async function checkIfBreachesExist(sha1, breaches) {
+  const breachResults = await HIBP.getBreachesForEmail(sha1, breaches, true);
+
+  if (breachResults.length >= 1) {
+    return true;
+  }
+
+  return false;
+}
+
+function getArgsValue(argument) {
+  const cliArguments = process.argv;
+
+  if (cliArguments.indexOf(argument) < 0) {
+    throw new Error(`You are missing the argument: ${argument}`);
+  }
+
+  const arguemntIndex = cliArguments.indexOf(argument);
+  const value = cliArguments[(arguemntIndex + 1)];
+
+  if (!value ) {
+    throw new Error(`No value set for ${argument}.`);
+  }
+
+  const valueNumber = parseInt(value);
+
+  if (Number.isNaN(valueNumber)) {
+    throw new Error(`The value for ${argument} is not an interger.`);
+  }
+
+  return valueNumber;
+
+}
+
+(async () => {
+  console.log("Script starting");
+
+  const allHibpBreachesResp = await HIBP.req("/breaches");
+  const allHibpBreaches = allHibpBreachesResp.body;
+
+  const limitQuery = getArgsValue("--limit");
+  const cohortSize = getArgsValue("--cohort-size");
+
+  console.log(`The limit of this query is ${limitQuery}`);
+  console.log(`The target cohort size of this query is ${cohortSize}`);
+
+  // "SELECT primary_email, primary_sha1 FROM subscribers WHERE signup_language LIKE 'en%' AND breaches_resolved IS NULL ORDER BY random();"
+
+  const results = await knex("subscribers").where("signup_language", "like", "en%").andWhere({breaches_resolved: null}).orderByRaw("RANDOM()").limit(limitQuery).select("primary_email", "primary_sha1");
```

Comment:
suuuper nit: Move `.select("primary_email, "primary_sha1")` after the `knex("subscribers")` to look more SQLey? Also might benefit from a couple line breaks to make it easier to parse.

```js
const results = await knex("subscribers")
  .select("primary_email", "primary_sha1")
  .where("signup_language", "like", "en%")
  .andWhere({breaches_resolved: null})
  .orderByRaw("RANDOM()")
  .limit(limitQuery);
```